app/assets/stylesheets/common.scss

```diff
@@ -1,3 +1,4 @@
+@import "common/variables";
```

Comment:
Can we rename this file, maybe `font_variables`? We already have `common/foundation/variables` and it would be nice to avoid the filename clash.

Reply:
Sure will do!